q2_diversity/_beta/_method.py

```diff
@@ -63,6 +76,31 @@ def beta_phylogenetic(table: biom.Table, phylogeny: skbio.TreeNode,
     return results
 
 
+def beta_phylogenetic_alt(table: BIOMV210Format, phylogeny: NewickFormat,
```

Comment:
Would it be possible to politely catch and err if the `table` has < 4 samples? That's an edge case we haven't handled yet behind the scenes

Reply:
I am not sure that this is the reason why we've observed failures, for all the unit tests we compute distances on a 2x2 sample matrix, and those tests pass.